scala_config.bzl

```diff
@@ -0,0 +1,36 @@
+load("@bazel_skylib//lib:versions.bzl", "versions")
+load("//scala:scala_cross_version.bzl", "extract_major_version")
+
+def _default_scala_version():
+    """return the scala version for use in maven coordinates"""
+    return "2.12.11"
+
+def _store_config(repository_ctx):
+    bazel_version = versions.get()
```

Comment:
Maybe replace `versions.get()` with `native.bazel_version`? This is what `skylib` does under the hood. As it is now it requires `skylib` to be declared before `scala_config`. Maybe not an issue though.

Reply:
Thanks, good point!